pages/portfolio.html

```diff
@@ -4,9 +4,121 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio | Julie Warren</title>
+  <link rel="stylesheet" href="../style.css" />
+  <!-- favicon -->
+  <link rel="apple-touch-icon" sizes="180x180" href="../assets/favicon/apple-touch-icon.png">
+  <link rel="icon" type="image/png" sizes="32x32" href="../assets/favicon/favicon-32x32.png">
+  <link rel="icon" type="image/png" sizes="16x16" href="../assets/favicon/favicon-16x16.png">
+  <link rel="manifest" href="../assets/favicon/site.webmanifest">
+  <link rel="mask-icon" href="../assets/favicon/safari-pinned-tab.svg" color="#087e8b">
+  <meta name="msapplication-TileColor" content="#2b5797">
+  <meta name="theme-color" content="#ffffff">
 </head>
 <body>
-  
+  <!-- navigation bar -->
+  <header class="portfolio-header">
+    <nav class="nav-container">
+      <a href="../index.html">
+        <img id="nav-logo" alt="logo of initials J A W enclosed in a circle" src="../assets/images/JAW.png">
+      </a>
+      <ul class="nav-buttons">
+        <li>
+          <a href="../index.html">Home</a>
+        </li>
+        <li>
+          <a href="./about.html">About</a>
+        </li>
+        <li>
+          <a href="portfolio.html">Portfolio</a>
+        </li>
+      </ul>
+    </nav>
+    <div class="break">
+    </div>      
+    <h1 class="page-header">
+        Technical Portfolio
+    </h1>
+    </nav>
+  </header>
+  <!-- main body -->
+  <main class="main-container">
+    <div class="project-grid">
+      <div class="icon icon1">
+        <img class="cats" alt="two cats sharing a too-small cat tower perch" src="../assets/images/cats.jpg">
+      </div>
+      <div class="text text1">
```

Comment:
Some of these classes, like `text1` seem like they're better suited to be an id, because they're used to identify just a single element. Also, a more descriptive name would be nice.